app/assets/javascripts/discourse/app/components/post/menu/buttons/admin.gjs

```diff
@@ -0,0 +1,24 @@
+import Component from "@glimmer/component";
+import DButton from "discourse/components/d-button";
+
+export default class PostMenuAdminButton extends Component {
+  static shouldRender(args) {
+    return (
+      args.post.canManage || args.post.can_wiki || args.post.canEditStaffNotes
+    );
+  }
+
+  <template>
+    {{#if @shouldRender}}
+      <DButton
+        class="post-action-menu__admin show-post-admin-menu"
+        ...attributes
```

Comment:
Similar to the previous comment - I don't think this is actually being used? Can we drop it in all the buttons for simplicity/readability?

Reply:
We can't remove it because on the buttons `...attributes` is being used to propagate the `btn-flat` class that is applied in the `PostMenuButtonWrapper` component